app/views/notifications/index.html.erb

```diff
@@ -68,6 +68,9 @@
     <%= render "articles/fitvids" %>
   </main>
   <%= javascript_packs_with_chunks_tag "followButtons", defer: true %>
+  <script>
+    <%== RunkitTag.script %>
+  </script>
```

Comment:
I've added this to prevent the unhandled error we're currently seeing when:

- You view a comment from the notificatons page
- Press reply, type a response and submit
- Error in the console: `activateRunKitTags` is not defined

This needed to be fixed as part of this PR as Cypress automatically fails when an unhandled error occurs